src/packages/cli/package.json

```diff
@@ -124,9 +124,9 @@
   },
   "scripts": {
     "test:sqlite": "jest integrate.sqlite",
+    "test:postgresql": "jest integrate.postgresql --forceExit",
```

Comment:
@timsuchanek had to resort to forceExit for postgresql integration tests. The near exact same code exits fine for sqlite. Thus Is suspect the thing keeping the node process running is the pg client. But I don't see how so, since we call `.end()`.

For now this works, but its ugly.